Flat/config/GeneralLeptonicTree.cfg

```diff
@@ -0,0 +1,142 @@
+# general
```

Comment:
I thought we were removing GeneralLeptonicTree from the repository

Reply:
Ok for me, for some reason I got it when I pull the version from git. Indeed, I don't use it